apps/android/app/src/main/java/chat/simplex/app/views/chat/group/AddGroupMembersView.kt

```diff
@@ -35,14 +34,16 @@ import chat.simplex.app.views.usersettings.SettingsActionItem
 fun AddGroupMembersView(groupInfo: GroupInfo, chatModel: ChatModel, close: () -> Unit) {
   val selectedContacts = remember { mutableStateListOf<Long>() }
   val selectedRole = remember { mutableStateOf(GroupMemberRole.Member) }
-
+  var allowModifyMembers by remember { mutableStateOf(true) }
   BackHandler(onBack = close)
   AddGroupMembersLayout(
     groupInfo = groupInfo,
     contactsToAdd = getContactsToAdd(chatModel),
     selectedContacts = selectedContacts,
     selectedRole = selectedRole,
+    allowModifyMembers = allowModifyMembers,
     inviteMembers = {
+      allowModifyMembers = false
```

Comment:
@avently is it only set to true when view appears? i.e. it will still be disabled even after all members are invited if the view is not closed for some reason, and so it's because the view is automatically dismissed once operation is complete we don't set it to true anywhere else? e.g. can we set it to true after member invitations for loop?

Reply:
Since we closing the view there is no need to set it to true. If we wouldn't close it, then yes, we would have to make it `true`